<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrivialIPClass</class>
 <widget class="QMainWindow" name="TrivialIPClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrivialIP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="3" column="3">
         <widget class="QLineEdit" name="lineEditScaleY"/>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="pushButtonScale">
          <property name="text">
           <string>缩放</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>x方向(倍数)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="5">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>790</width>
             <height>260</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="image"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="0" colspan="5">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Width:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="width_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="height_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="5">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>R:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="red_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>G:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="green_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>B:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="blue_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEditScaleX"/>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>y方向(倍数)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="5">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>旋转度数(单位:°)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditRotation"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButtonRotation">
            <property name="text">
             <string>旋转</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButtonMirror">
            <property name="text">
             <string>镜像</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0" colspan="5">
         <layout class="QGridLayout" name="gridLayout_4"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBoxMosaic">
          <property name="text">
           <string>打马赛克(直接在图片上滑动)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonHist">
          <property name="text">
           <string>直方图均衡化</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonBack">
          <property name="text">
           <string>后退</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonSave">
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButtonForward">
          <property name="text">
           <string>前进</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开图片</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="trivialip.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>image</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonBack</sender>
   <signal>clicked()</signal>
   <receiver>image</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonForward</sender>
   <signal>clicked()</signal>
   <receiver>image</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>image</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxMosaic</sender>
   <signal>clicked(bool)</signal>
   <receiver>image</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
